<ui version="4.0" >
 <class>preferencesDialog</class>
 <widget class="QDialog" name="preferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ElmerGUI.qrc" >:/icons/Mesh3D.png</iconset>
  </property>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>320</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Ok</string>
   </property>
   <property name="icon" >
    <iconset resource="../ElmerGUI.qrc" >:/icons/dialog-ok-apply.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>320</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="icon" >
    <iconset resource="../ElmerGUI.qrc" >:/icons/dialog-close.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="featureGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Feature edges</string>
   </property>
   <widget class="QRadioButton" name="surfaceRButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Use surface mesh</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="volumeRButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Use volume mesh</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angleSpin" >
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>0</number>
    </property>
    <property name="maximum" >
     <number>90</number>
    </property>
    <property name="value" >
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="angleLabel" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Angle (deg):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lineWidthSpin" >
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>50</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lineWidthLabel_2" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Line width:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyButton" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>320</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="meshEdgesGroup" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Mesh edges</string>
   </property>
   <widget class="QRadioButton" name="meshEdgesSurface" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Use surface mesh</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="meshEdgesVolume" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Use volume mesh</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="meshLineWidth" >
    <property name="geometry" >
     <rect>
      <x>340</x>
      <y>50</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="meshLineWidthLabel" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Line width:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
